mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenProjectConfigurator.java

```diff
@@ -47,11 +46,32 @@ void addRequiredDependencies(Model model) {
                 final Dependency dependency = dependencies.get(strategy);
                 model.addDependency(dependency);
             }
+            addSurefireApiDependency(model);
         } catch (IOException e) {
             throw new RuntimeException("Unable to load strategy definitions", e);
         }
     }
 
+    private void addSurefireApiDependency(Model model) {
+        if (surefireVersion != null && surefireVersion.toString() != null) {
+            boolean alreadyContains = model.getDependencies().stream()
+                .anyMatch(dep ->
+                    "org.apache.maven.surefire".equals(dep.getGroupId()) && "surefire-api".equals(dep.getArtifactId()));
+            if (!alreadyContains) {
+                model.addDependency(getSurefireApiDependency(surefireVersion.toString()));
+            }
+        }
+    }
+
+    private Dependency getSurefireApiDependency(String version){
```

Comment:
Funny, I had the same idea when I was writing the code, but I was too lazy to create a new class just because of few lines of code :-D.